pkgs/data/themes/catppuccin-kde/default.nix

```diff
@@ -1,62 +1,89 @@
-{ lib
-, stdenvNoCC
-, fetchFromGitHub
-, fetchpatch
-, flavour ? [ "frappe" ]
-, accents ? [ "blue" ]
-, winDecStyles ? [ "modern" ]
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  fetchpatch,
+  flavour ? [ "frappe" ],
+  accents ? [ "blue" ],
+  winDecStyles ? [ "modern" ],
 }:
-
 let
-  validFlavours = [ "mocha" "macchiato" "frappe" "latte" ];
-  validAccents = [ "rosewater" "flamingo" "pink" "mauve" "red" "maroon" "peach" "yellow" "green" "teal" "sky" "sapphire" "blue" "lavender" ];
-  validWinDecStyles = [ "modern" "classic" ];
+  validFlavours = [
+    "mocha"
+    "macchiato"
+    "frappe"
+    "latte"
+  ];
+  validAccents = [
+    "rosewater"
+    "flamingo"
+    "pink"
+    "mauve"
+    "red"
+    "maroon"
+    "peach"
+    "yellow"
+    "green"
+    "teal"
+    "sky"
+    "sapphire"
+    "blue"
+    "lavender"
+  ];
+  validWinDecStyles = [
+    "modern"
+    "classic"
+  ];
 
   colorScript = ./color.sh;
 in
 
-  lib.checkListOfEnum "Invalid accent, valid accents are ${toString validAccents}" validAccents accents
-  lib.checkListOfEnum "Invalid flavour, valid flavours are ${toString validFlavours}" validFlavours flavour
-  lib.checkListOfEnum "Invalid window decoration style, valid styles are ${toString validWinDecStyles}" validWinDecStyles winDecStyles
-
-stdenvNoCC.mkDerivation rec {
-  pname = "kde";
-  version = "0.2.4";
-
-  src = fetchFromGitHub {
-    owner = "catppuccin";
-    repo = pname;
-    rev = "v${version}";
-    hash = "sha256-w77lzeSisx/PPxctMJKIdRJenq0s8HwR8gLmgNh4SH8=";
-  };
-
-  patches = [
-    (fetchpatch {
-      url = "https://github.com/michaelBelsanti/catppuccin-kde/commit/81a8edb3c24bd6af896c92b5051e09af97d69c51.patch";
-      hash = "sha256-cb4/dQ52T+H8UqXEgExblmnMfxwO0Y1BrjMCay/EAkI=";
-    })
-  ];
+lib.checkListOfEnum "Invalid flavour, valid flavours are ${toString validFlavours}" validFlavours flavour
+lib.checkListOfEnum "Invalid accent, valid accents are ${toString validAccents}" validAccents accents
+lib.checkListOfEnum "Invalid window decoration style, valid styles are ${toString validWinDecStyles}" validWinDecStyles winDecStyles
 
-  installPhase = ''
-    runHook preInstall
-    patchShebangs .
+stdenvNoCC.mkDerivation
+  rec {
+    pname = "kde";
```

Comment:
TBH this should be rectified to be `catppuccin-kde` like all the other Catppuccin themes — I know it's been here for a while but now's the time to do it, I think

Reply:
I was also considering this. I'll have a look at how #294507 handled the script and work on a version with the "master" catppuccin theme. If anyone has any input on a potential preferred method, I think it would be a helpful discussion